src/Dependencies/.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*.cs]
+
+# IDE0240: Remove redundant nullable directive
+# The directive needs to be included since all sources in a source package are considered generated code
+# when referenced from a project via package reference.
+dotnet_diagnostic.IDE0240.severity = none
```

Comment:
I would have assumed we would just set the generated_source flag here, but I guess is your idea that we still get validation for the code in our own build, and the package itself won't enforce anything? If so, maybe add a comment to that effect as a mop-up?

Reply:
There are two editorconfigs in play: one is included in the packages themselves, which ensures the code is treated as generated in the repos that include the packages, the other is here with suppresses redundant nullable directive warnings reported because the analyzer doesn't understand that `#nullable enable` directive is required in files marked as generated.

Reply:
OK that matches my understanding. Not sure if we should put a comment in this one before somebody "fixes" this.